contracts/modules/cover/Cover.sol

```diff
@@ -63,9 +62,9 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
   uint private constant GLOBAL_CAPACITY_DENOMINATOR = 10_000;
   uint private constant REWARD_DENOMINATOR = 10_000;
 
-  uint public constant MAX_COVER_PERIOD = 364 days;
+  uint private constant MAX_COVER_PERIOD = 365 days;
```

Comment:
why is there a value increase here?

Doesn't this affect the bucket size computation with 91 * 4 = 364?

Reply:
Most of the time we'll be mid-tranche (not bucket). This means that we'll use 4 tranches for a 1-year cover anyway so the 364 days max only helps in very few cases.

Reply:
sorry meant to say tranche, ok understood